resources/migrations/009-add-item-field-constraint.edn

```diff
@@ -0,0 +1,6 @@
+;; resources/migrations/009-add-item-field-constraint.edn
+{:up   ["ALTER TABLE field_values
+         ADD CONSTRAINT item_field_unique
```

Comment:
This would disallow repeated fields. Is that alright?

Reply:
This constraints a single value for a field of an item ex. Book no. 6 can have only one ISBN number. Multiple fields called ISBN number would be allowed currently.